Networking/Networking/Mapper/ProductVariationMapper.swift

```diff
@@ -28,6 +28,33 @@ struct ProductVariationMapper: Mapper {
     }
 }
 
+/// Mapper: ProductVariations
+///
+struct ProductVariationsMapper: Mapper {
```

Comment:
I think this name is a little bit confusing, especially because we already have a `ProductVariationListMapper`.
What do you think about a `ProductVariationsBulkUpdateMapper`, and I think it can exist in its own file too.

Reply:
Yes also latter we could extend it to add bulk create and bulk delete I updated the PR.